docs/source/using/host-folders.adoc

```diff
@@ -0,0 +1,225 @@
+<4> (Required) The username for the CIFS share.
+<5> (Required) The password for the CIFS share.
+<6> The domain of the share. Often this can be left blank, but for example on Windows, when your account is linked to a Microsoft account, you must use the Microsoft account email address as username as well as your machine name as displayed by _$env:COMPUTERNAME_ as domain.
+
+TIP: On Windows hosts the `hostfolder add` command also provides a _users-share_ option.
+When specified no UNC path needs to be specified and the _C:\Users_ is assumed.
+
+WARNING: When you use the Boot2Docker ISO along with the VirtualBox driver, VirtualBox guest additions are automatically enabled and occupy the _/Users_ mountpoint.
+
+==== Instance specific host folders
+
+Per default host folder definitions are persistent similar to options specified using the `<<managing-minishift#persistent-configuration, minishift config>>` command.
+This means that these host folder definitions will survive the deletion and subsequent re-creation of a Minishift VM.
+
+In some cases you might want to define a host folder just for a specific Minishift instance. To do so you can use the _instance-only_ flag of the link:../command-ref/minishift_hostfolder_add{outfilesuffix}[`hostfolder add`] command.
+Host folder definition created with the _instance-only_ flag will be removed together with any other instance specific state during a `<<managing-minishift#minishift-delete,minishift delete>>`.
+
+[[mounting-host-folders]]
+=== Mounting host folders
+
+Once you have added your host folder definitions (see <<adding-cifs-hostfolder>>), the link:../command-ref/minishift_hostfolder_mount{outfilesuffix}[`hostfolder mount`] command allows you to mount a host folder by its name:
+
+----
+$ minishift hostfolder mount mac
+Mounting 'mac': '//192.168.99.1/MAC' as '/mnt/sda1/mac' ... OK
+----
+
+You can verify that the host folder is mounted by running:
+
+----
+$ minishift hostfolder list
+Name   Mountpoint      Remote path          Mounted
+mac    /mnt/sda1/mac   //192.168.99.1/MAC   Y
+----
+
+Alternatively you can list the actual content of the mounted host folder:
+
+----
+$ minishift ssh "ls -al /mnt/sda1/mac"
+----
+
+[[auto-mounting-host-folders]]
+==== Auto-mounting of host folders
+
+Host folders can also be mounted automatically on each `<<managing-minishift#minishift-stop,minishift start>>`.
+To achieve this you need to set the <<managing-minishift#persistent-configuration,persistent configuration>> option _hostfolders-automount_:
+
+----
+$ minishift config set hostfolders-automount true
+----
+
+Once _hostfolders-automount_ is set, Minishift will attempt to mount all defined host folders during `minishift start`.
+
+[[umounting-host-folders]]
+=== Unmounting host folders
+
+Opposite to the `hostfolder add` command, link:../command-ref/minishift_hostfolder_umount{outfilesuffix}[`hostfolder umount`] can be used to unmount a host folder:
+
```

Comment:
Opposite to>In contrast to
... hostfolder umount can be used> the hostfolder umount command can be used